cmake/onnxruntime_external_ops.cmake

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
rename this file to onnxruntime_language_interop_ops.cmake which in turn calls ..python.cmake. the language interop cmake files can go in a separate folder under 'cmake'. idea is to make it easier to add ops in other languages following this framework.

Reply:
Done.